sa/saro.go

```diff
@@ -443,10 +454,13 @@ func (ssa *SQLStorageAuthorityRO) GetOrder(ctx context.Context, req *sapb.OrderR
 			return nil, err
 		}
 
+		idents := make([]identifier.ACMEIdentifier, 0, len(authzValidityInfo))
 		names := make([]string, 0, len(authzValidityInfo))
 		for _, a := range authzValidityInfo {
+			idents = append(idents, identifier.ACMEIdentifier{Type: identifier.IdentifierType(uintToIdentifierType[a.IdentifierType]), Value: a.IdentifierValue})
```

Comment:
can `unitToIdentifierType` just straightforwardly return an `identifier.IdentifierType`, rather than returning a string which we then have to convert to that type?